test/e2e/density.go

```diff
@@ -111,6 +114,24 @@ var _ = Describe("Density", func() {
 		}
 		itArg := testArg
 		It(name, func() {
+
```

Comment:
This block gets longer and longer. Maybe we could move this block to a separate function?